File: CompanyTaskManager/Models/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CompanyTaskManager.Models
{
    public class Task
    {
        public int TaskId { get; set; }

        public int WorkplacementId { get; set; }
        public Workplacement Workplacement { get; set; }

        public int EmployeeId { get; set; }
        public User Employee { get; set; }

        public int AddedById { get; set; }
        public User AddedBy { get; set; }

        public String Title { get; set; }
        public String Description { get; set; }
        public String Priority { get; set; }
        public String Status { get; set; }
        public DateTime AddDate { get; set; }

    }
}

File: CompanyTaskManager/ClientApplication/company-task-manager/src/app/models/Task.ts
import { Workplacement } from './Workplacement';
import { User } from './User'

export interface Task{
    taskId: number
    title: string,
    description: string,
    priority: string, 
    employeeId: number,
    employee: User,
    addedById: number,
    addedBy: User,
    workplacementId: number,
    workplacement: Workplacement,
    addDate: Date
    status: string
}